### DISCUSSION TITLE
Welcome to python-satcfdi Discussions!

### DISCUSSION THREAD

**Discussion by satcfdi** [SAT-CFDI/python-satcfdi#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zalvatore** [comment]
Ena la descarga de constancia fiscal, cual es el valor para id_cif??

`res = csf.retrieve("AAA010101AAA", id_cif="012345678")`

**Reply by satcfdi** [reply]
El valor se encuentra en la constacia de situacion fiscal

**Comment by TotalConta** [comment]
Tengo una duda he leido la documentacion del SAT, pero quiero preguntar por si no lo vi, ¿es posible recuperar los id de solicitud? me refiero a este id_solicitud='365e2a3d-6f99-4563-856e-28caddc7ad39',
